modules/core/src/file-utils/load-image.js

```diff
@@ -0,0 +1,23 @@
+import {getPathPrefix} from './path-prefix';
+/* global Image */
+
+/*
+ * Loads images asynchronously (browser only)
+ * image.crossOrigin can be set via options.crossOrigin, default to 'anonymous'
+ * returns a promise tracking the load
+ */
+export function loadImage(url, options) {
```

Comment:
This does not work in a web worker. See the patch on luma: https://github.com/uber/luma.gl/blob/6.4-release/modules/core/src/io/browser-load.js#L32

Reply:
Good point, moved the URL handling into the load method and the image handling into the image-utils and loaders.